app/src/features/apiClient/screens/apiClient/components/sidebar/components/collectionsList/requestRow/RequestRow.tsx

```diff
@@ -13,13 +13,16 @@ import { useSelector } from "react-redux";
 import { getUserAuthDetails } from "store/slices/global/user/selectors";
 import { getCurrentlyActiveWorkspace } from "store/features/teams/selectors";
 import { toast } from "utils/Toast";
+import { MoveToCollectionModal } from "../../../../modals/MoveToCollectionModal/MoveToCollectionModal";
 
 interface Props {
   record: RQAPI.ApiRecord;
 }
 
 export const RequestRow: React.FC<Props> = ({ record }) => {
   const [isEditMode, setIsEditMode] = useState(false);
+  const [isMoveToCollectionModalOpen, setIsMoveToCollectionModalOpen] = useState(false);
+  const [requestToMove, setRequestToMove] = useState(null);
```

Comment:
can we make this one state? `requestToMoveToCollection` and if it is present, show modal?